lca-chimevc-stack/lambda_functions/chime_call_analytics_initialization/index.js

```diff
@@ -599,7 +599,10 @@ const handler = async function handler(event, context) {
       callData = await getCallDataFromChimeEventsWithLambdaHook(event);
 
       console.log('Saving callData to DynamoDB');
-      await writeCallDataToDdb(callData);
+      await writeCallDataToDdb(callData, callData.callId);
```

Comment:
Would be good to add a comment to say why we write the callData twice when callId != originalCallId..  It's not obvious why we want to do that, instead of doing it in an if/else - and without an explanation it would be easy for someone to think the are 'cleaning up the code' and accidentally break the logic.